src/common/protoc-executor.ts

```diff
@@ -43,17 +43,26 @@ namespace ProtocExecutor {
 
     const mount_dirname = '/opt/protoc';
     const mounted_proto_path = path.posix.join(mount_dirname, ServiceConfig.convertServiceNameToFolderName(dependency_config.name), dependency_config.proto);
-    execSync([
-      'docker', 'run',
-      '-v', `${target_path}:/defs`,
-      '-v', `${tmpRoot}:${mount_dirname}`,
-      '--user', process.platform === 'win32' ? '1000:1000' : '$(id -u):$(id -g)',  // TODO figure out correct user for windows
-      'architectio/protoc-all',
-      '-f', `${mounted_proto_path}`,
-      '-i', mount_dirname,
-      '-l', target_language,
-      '-o', MANAGED_PATHS.DEPENDENCY_STUBS_DIRECTORY
-    ].join(' '), {stdio: 'ignore'});
+
+    // execSync caused the output logs to hang
+    await new Promise(resolve => {
```

Comment:
Why this change if we’re just going to await the result anyway?

Reply:
execSync was causing the spinners to hang for whatever reason even if it was in an async func. For whatever reason await exec causes it to behave as expected.